bicycleparameters/cycle_app.py

```diff
@@ -213,6 +221,30 @@
 
 # creates generic set of Benchmark parameters
 
+@app.callback(Output('download-button', 'href'),
+              [Input('bike-dropdown', 'value'),
+               Input('range-slider', 'value')])
+
+def update_download_link(value,slider):
+    bicycle = bp.Bicycle(value, pathToData=path_to_app_data)
```

Comment:
It seems this is likely to return the eigenvalues from one of the default bicycle selections, but what if the user changes the table values? It isn't clear to me that you would download the correct eigenvalues if the table is changed.